2017/JULY/LTIME50/MAXOR.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+	Made by:
+		``` Monal Shadi ``
+	Github: Monal5031
+	Codechef: monalshadi
+*/
+
+#include "bits/stdc++.h"
+
+using namespace std; 
```

Comment:
Remove Extra Spaces @Monal5031

Reply:
@Lucifer24 I think we were focusing more on code now rather than spacings, if you still wish I'll make the changes.

Reply:
@Monal5031 At first i wasn't expecting these things in your code and secondly we made @PiyushPawar17 correct his spaces too so i guess you can do it too

Reply:
@aashutoshrathi Please stop misusing your write access by editing other people's comments.
@Lucifer24 We cannt go on doing changes as it were done in one of the PRs, we need to follow some standard guideline.